### DISCUSSION TITLE
How to get SPSS Statistics to work whitin SEB?

### DISCUSSION THREAD

**Discussion by jpmaroco** [SafeExamBrowser/seb-mac#196] [Q&A]
Dear all,
After reading the manuals for both Win and OS X versions of SEB 3.3 Configuration Tool, I can't get it to find and run the SPSS Statistics (stats.exe for windows; SPSS Statistics.app for OSX). Here is what I did:

First just typed "stats.exe" in the executable box for Win, it did not work (i.e., did not find the stats.exe, altough SPSS statistics was installed with default options and I can run in the cmd box). It did work when I added the full path:

![image](https://user-images.githubusercontent.com/89992415/157671802-e458a8a3-c79c-4586-9cda-34a23d8c3f8c.png)

For the Mac OS X I also just added the "SPSS Statistics.app" to the Executable box. Did not worked.
2.1. Following the example from Windows, I added the full path:
![image](https://user-images.githubusercontent.com/89992415/157671864-746e5537-afeb-46b9-9e47-be05eb25f9c4.png)

It does not work either.

Is this a bug? or am I doing something wrong with the Mac Configuration?

Thanks for any insights on how to get the SPSS Statistics app running with SEB.

**Comment by danschlet** [comment]
As the [manual of SEB for macOS](https://safeexambrowser.org/macosx/mac_usermanual_en.html#PermittedProcessesSection) states:
Using permitted processes is not supported in SEB for macOS. You can define and edit permitted processes for SEB for Windows here.

Apple just added some API in macOS 12 which will allow to run additional applications when using the AAC kiosk mode. We will support this in SEB for macOS at some point, although right now it may not be very helpful as it will only work for students which use macOS Monterey.

**Reply by danschlet** [reply]
If you need to run complex third party applications in a really secure way, we anyways recommend to use a virtual desktop environment like VMware Horizon (see [this paper](http://safeexambrowser.org/vdi/Competence-Oriented_Exams_VDI+SEB_ETH-LET_2019_05.pdf) about the environment we are using) or any of the cloud based from Microsoft or Amazon. The idea is to secure the local machine with SEB and connect to the secured virtual desktop with a full screen browser window even from Macs. The desktops itself can only run Windows or Linux for legal reasons (Apple doesn't allow virtualization of macOS on a non-Mac server).

**Reply by jpmaroco** [reply]
Thank you @danschlet  very much for your repply. I was afraid I was doing something not right. But now I am puzzled why SEB, in the Application tabs has a OS X option (just like for Windows) if it can't work.... 
Warm regards,

**Reply by danschlet** [reply]
We basically added the option ahead of it being actually supported, when we thought to be able to implement support for that feature and then didn't remove it when the roadmap changed. It's still an open source project with some pitfalls...

**Reply by jpmaroco** [reply]
And It is a great open source project. It works great in Windows and I am thankfull for it (to bad the Apple policies).
